webextension/background.js

```diff
@@ -1,3 +1,63 @@
+browser.contextMenus.create({
+  id: "open-in-this-container",
+  title: "Always Open in This Container",
+  contexts: ["all"],
+});
+
+function getUserContextIdFromCookieStore(cookieStore) {
+  const container = cookieStore.replace("firefox-container-", "");
+  if (container !== cookieStore) {
+    return container;
+  }
+  return false;
+}
+
+const storagePrefix = "originMap@@_";
+const storageArea = browser.storage.local;
+
+browser.contextMenus.onClicked.addListener((info, tab) => {
```

Comment:
Yeah if we add any certainly. There is a click handler per each menu item too, I just didn't test that out yet.